### PR TITLE
fix(chat): fix marketplace counter (Issue #2326)

### PR DESCRIPTION
**Pull request by Derikyan**
**Description:**

Hide the counter in all places in marketplace

Issues:

- Issue #2326 

**UI changes**

Снимок экрана 2024-10-09 в 16 11 03

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2337.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — CHANGES_REQUESTED** [review]
We shouldn't remove this functional

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2337.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Marketplace/SearchHeader.tsx [hunk 1]
```diff
@@ -23,10 +23,10 @@ import ContextMenu from '@/src/components/Common/ContextMenu';
 
 import { Feature } from '@epam/ai-dial-shared';
 
-const countLabel = {
-  [MarketplaceTabs.HOME]: 'DIAL Marketplace',
-  [MarketplaceTabs.MY_APPLICATIONS]: 'My workspace',
-};
+// const countLabel = {
+//   [MarketplaceTabs.HOME]: 'Home page',
+//   [MarketplaceTabs.MY_APPLICATIONS]: 'My applications',
```
Comment (IlyaBondar) [inline apps/chat/src/components/Marketplace/SearchHeader.tsx]: revert names

**Review by IlyaBondar — APPROVED** [review]

File: apps/chat/src/components/Marketplace/SearchHeader.tsx (hunk 1 above)
Comment (Derikyan) [reply apps/chat/src/components/Marketplace/SearchHeader.tsx]: done

**Pull request merged**